tests/testthat/test-queuefallout.R

```diff
@@ -1,24 +1,49 @@
 test_that("Validate QueueFallout using legacy credentials", {
-  
+
   skip_on_cran()
-  
+
   #Correct [masked] credentials
   SCAuth(Sys.getenv("USER", ""), Sys.getenv("SECRET", ""))
-  
-  aa <- QueueFallout("zwitchdev", 
-                     "2014-12-01", 
-                     "2015-12-31", 
-                     metric="pageviews", 
-                     element="page", 
+
+  aa <- QueueFallout("zwitchdev",
+                     "2014-12-01",
+                     "2015-12-31",
+                     metric="pageviews",
+                     element="page",
                      c("http://randyzwitch.com/r-amazon-ec2", "http://randyzwitch.com/julia-import-data")
   )
-  
+
   #Validate returned value is a data.frame
   expect_is(aa, "data.frame")
-  
+
 })
```

Comment:
You have extra `})` values in each function call in the QueueFallout file. There should just be one at the end representing the closing of the `{` bracket in the `test_that` function